client/src/components/Button.jsx

```diff
@@ -0,0 +1,17 @@
+import PropTypes from "prop-types";
+
+function Button({ buttonHandler, className, innerText }) {
```

Comment:
it make more sense (and it's a standard) to call it onClick
```suggestion
function Button({ onClick, className, innerText }) {
```